Update dbt_project.yml.md (#5545)
fishtown-analytics/docs.getdbt.com: website/docs/reference/dbt_project.yml.md

Every [dbt project](/docs/build/projects) needs a `dbt_project.yml` file — this is how dbt knows a directory is a dbt project. It also contains important information that tells dbt how to operate your project.

- dbt uses [YAML](https://yaml.org/) in a few different places. If you're new to YAML, it would be worth learning how arrays, dictionaries, and strings are represented.

<VersionBlock lastVersion="1.4">

- By default, dbt looks for the `dbt_project.yml` in your current working directory and its parents, but you can set a different directory using the `--project-dir` flag.

</VersionBlock>

<VersionBlock firstVersion="1.5">

- By default, dbt looks for the `dbt_project.yml` in your current working directory and its parents, but you can set a different directory using the `--project-dir` flag or the `DBT_PROJECT_DIR` environment variable.

> - Specify your dbt Cloud project ID in the `dbt_project.yml` file using `project-id` under the `dbt-cloud` config. Find your project ID in your dbt Cloud project URL: In example `https://cloud.getdbt.com/11/projects/123456`, the project ID is `123456`.

Reviewer: ```suggestion
- Specify your dbt Cloud project ID in the `dbt_project.yml` file using `project-id` under the `dbt-cloud` config. Find your project ID in your dbt Cloud project URL: For example, in `https://cloud.getdbt.com/11/projects/123456`, the project ID is `123456`.
```

</File>

</VersionBlock>

## Naming convention

It's important to follow the correct naming conventions for the YAML configs in your `dbt_project.yml` file to make sure dbt can understand them properly. This is especially true for resource types with more than one word:

- Use dashes (`-`) when configuring resource types with multiple words in your `dbt_project.yml` file:

  <File name="dbt_project.yml" >
  
    ```yaml

>     [saved-queries](/docs/build/saved-queries#project-level-saved-queries): ## Uses dashes in the dbt_project.yml file.

Author: ```suggestion
    [saved-queries](/docs/build/saved-queries#project-level-saved-queries): ## Use dashes in the dbt_project.yml file.
```

>     [saved-queries](/docs/build/saved-queries#project-level-saved-queries): ## This resource type in the dbt_project.yml uses dashes 

Author: ```suggestion
    [saved-queries](/docs/build/saved-queries#project-level-saved-queries): ## Uses dashes in the dbt_project.yml file.
```

      my_saved_query:
        config:
          +cache:
            enabled: true
    ```
  </File>

- Use underscore (`_`) when configuring resource types with multiple words outside the `dbt_project.yml` file:

  <File name="models/semantic_models.yml" >
  
    ```yaml

>     [saved_queries](/docs/build/saved-queries#configure-saved-query): ## This example uses underscores 

Author: ```suggestion
    [saved_queries](/docs/build/saved-queries#configure-saved-query): ## Use underscores everywhere outside the dbt_project.yml file.
```

</File>

</VersionBlock>

## Naming convention

> It's important to follow the correct naming conventions for the YAML configs in your `dbt_project.yml` file to make sure dbt can understand them properly. This is especially true for resource types with more than one word.

Reviewer: ```suggestion
It's important to follow the correct YAML naming conventions for the configs in your `dbt_project.yml` file to ensure dbt can process them properly. This is especially true for resource types with more than one word.
```

- Use dashes (`-`) when configuring resource types with multiple words in your `dbt_project.yml` file. Here's an example for [saved queries](/docs/build/saved-queries#configure-saved-query):

    <File name="dbt_project.yml">

    ```yml

>     saved-queries:  # Use dashes in the dbt_project.yml file.

Reviewer: ```suggestion
    saved-queries:  # Use dashes for resource types in the dbt_project.yml file.
```

      my_saved_query:
        config:
          +cache:
            enabled: true
    ```
    </File>

> - Use underscore (`_`) when configuring resource types with multiple words outside the `dbt_project.yml` file:

Reviewer: ```suggestion
- Use underscore (`_`) when configuring resource types with multiple words for YAML files other than the `dbt_project.yml` file. For example, here's the same saved queries resource in the `semantic_models.yml` file:
```
